tests/test_generator.py

```diff
@@ -550,3 +551,28 @@ def test_oneof_with_comments():
     )
 
     assert result == expected
+
+
+def test_generate_service_with_numeric_option():
+    PROTO_TEXT = """
+    syntax = "proto3";
+    package test.v1.proto_1;
+    service TestService {
+        option (test.options.v1.lifecycle) = DEPRECATED;
+        option (test.options.v1.major_version) = 1;
+    }
+    """
+
+    parsed_ast = Parser().parse(PROTO_TEXT)
+    result = Generator().generate(parsed_ast)
+
+    expected = (
+        'syntax = "proto3";\n'
+        "package test.v1.proto_1;\n"
+        "service TestService {\n"
+        '  option (test.options.v1.lifecycle) = "DEPRECATED";\n'
```

Comment:
The expected value DEPRECATED should be without the quotes here

Reply:
Should it? It's a string. I'm not familiar with this syntax. How can I tell that DEPRECATED should not be quoted? Are there keywords?

Reply:
I guess it's coming from `test/options/v1/option_extensions.proto`... Hmm.

Reply:
Ah, gotcha. So, the issue is that right now the parser treats all options as strings (and then does an int/bool check):

```python
    def visitOptionDecl(self, ctx: ProtobufParser.OptionDeclContext):
        name = self._getText(ctx.optionName())
        value = self._stringToType(self._getText(ctx.optionValue()))
        return ast.Option(name=name, value=value)
```

To make this work, I think we need to go farther and visit each option value. Let me experiment with this...

Reply:
One issue right now is that we don't parse nested .proto files. So if you have an import (as #33 does in the original example), we parse that in the AST as an `ast.Import(). We don't go inside, so we don't see an enum that would be defined inside. 🤔

Reply:
OTOH, if you've manually set the option value to be an `ast.Enum`, the generator should handle that..

Reply:
I think the parser doesn't really need to know it's an enum, just that it's not a string - so even without parsing the import that defines the enum the ast should be able to contain information that it's "something that doesn't need quotes"  ?

Reply:
Yep. Workin' on it.